sparkles/tests/test_find_er_catalog.py

```diff
@@ -0,0 +1,145 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import warnings
+from proseco import get_aca_catalog
+from proseco.tests.test_common import mod_std_info
+import numpy as np
+from Quaternion import Quat
+import Ska.Sun
+
+from sparkles.find_er_catalog import (
+    get_candidate_stars, find_er_catalog, filter_candidate_stars_on_ccd,
+    get_guide_counts, init_quat_from_attitude)
+
```

Comment:
I assume this also needs something like the `os.environ[agasc.SUPPLEMENT_ENABLED_ENV] = 'False'` that is used in test_review.py .